controllers/users.js

```diff
@@ -315,6 +315,35 @@ const verifyUserImage = async (req, res) => {
   }
 };
 
+/**
+ * Patch Update user nickname
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const updateDiscordUserNickname = async (req, res) => {
+  try {
+    const userToBeUpdated = await dataAccess.retrieveUsers({ id: req.params.userId });
+    const { discordId, username: userName } = userToBeUpdated.user;
+    if (!discordId) {
+      throw new Error("user not verified");
+    }
+    const response = await setUserDiscordNickname(userName, discordId);
+
+    return res.json({
+      userAffected: {
+        userId: req.params.userId,
+        username: userName,
```

Comment:
why making `username` as `userName` when it is being used as it is by default?

Reply:
I thought about changing it's casing into camelCase, But we actually don't need to change it.
reverted it back to `username`